src/components/fileviewer.js

```diff
@@ -242,31 +225,14 @@ const Line = ({ lineNumber, lineText, coverage, selectedLine, onLineClick }) =>
 
 /* This component contains metadata of the file */
 const FileViewerMeta = ({ revision, path, status, coverage }) => {
-  const showStatus = (label, fetched) => {
-    let msg;
-    if (fetched === undefined) {
-      msg = 'Fetching...';
-    } else if (fetched === true) {
-      msg = <span>&#x2714;</span>; // heavy checkmark
-    } else if (fetched === false) {
-      msg = <span>&#x2716;</span>; // heavy multiplication x
-    }
-    return (<li className="file-meta-li">{label}: {msg}</li>);
-  };
-
   return (
     <div className="file-meta-viewer">
       <div className="file-meta-center">
         <div className="file-meta-title">File Coverage</div>
         <CoveragePercentageViewer
           coverage={coverage}
         />
-        <div className="file-meta-status">
-          <ul className="file-meta-ul">
-            { showStatus('Source code', status.fetch.source) }
-            { showStatus('Coverage', status.fetch.coverage) }
-          </ul>
-        </div>
+        <status/>
```

Comment:
The most important (but not working) line.  The `status` is an instance of `FileViewerStatus`, and I want to  call it's render method with `<status/>`.